src/index.html

```diff
@@ -3,11 +3,261 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>The MET</title>
+    <link
+      rel="shortcut icon"
+      href="./images/photos/THEMET.png"
+      type="image/png">
+    <!-- <link rel="stylesheet" href="./styles/main.scss"> -->
```

Comment:
yet again - we seemed to resolve this, haven't we?
use `.scss`

Reply:
fixed